example/screens/RoomScreen.tsx

```diff
@@ -120,7 +129,7 @@ const RoomScreen = ({ navigation }: Props) => {
           accessibilityLabel={SWITCH_CAMERA_BUTTON}
         />
         <InCallButton
-          iconName={isScreencastOn ? 'share-off' : 'share'}
+          iconName={isScreencastOn ? 'share' : 'share-off'}
```

Comment:
I think that for screensharing icons should be swapped. If screencast is off we suggest to share, if it is on we suggest to stop sharing.

Reply:
Idk. I've tested how it looks on other tools, probably we might need to use different icon set for sharing (instead of justo on/off). I would leave is as is for now, and improve it when we decide to visually update app.